extensions/README.md

```diff
@@ -97,22 +98,9 @@ extensions, Khronos extensions, or inclusion in a future version of the glTF spe
 |-----------|--------|
 | [KHR_animation_pointer](https://github.com/KhronosGroup/glTF/pull/2147) | Ready for testing. |
 | [KHR_audio](https://github.com/KhronosGroup/glTF/pull/2137) | Ready for testing. |
-| [KHR_materials_anisotropy](https://github.com/KhronosGroup/glTF/pull/1798) | Ready for testing. |
 | [KHR_materials_diffuse_transmission](https://github.com/KhronosGroup/glTF/pull/1825) | Ready for testing. |
 | [KHR_materials_sss](https://github.com/KhronosGroup/glTF/pull/1928) | In development. |
 
-## Extensions for glTF 1.0
-
-### Khronos extensions for glTF 1.0
-
-* [KHR_binary_glTF](1.0/Khronos/KHR_binary_glTF/README.md)
-* [KHR_materials_common](1.0/Khronos/KHR_materials_common/README.md)
-
-### Vendor extensions for glTF 1.0
-
-* [CESIUM_RTC](1.0/Vendor/CESIUM_RTC/README.md)
-* [WEB3D_quantized_attributes](1.0/Vendor/WEB3D_quantized_attributes/README.md)
-
```

Comment:
Why was this section removed? This readme file technically applies to both 1.0 and 2.0 subfolders.

Reply:
I think it's time to stop calling attention to the 1.0 folders.  The folders still exist, so if any legacy users still need access, they can look in those folders as needed.  But I'm certain that Khronos no longer needs to call them out in a registry.